cl/users/models.py

```diff
@@ -174,6 +180,69 @@ class Meta:
         verbose_name_plural = "user profiles"
 
 
+class EmailFlag(models.Model):
+    BAN = "ban"
+    FLAG = "flag"
+    FLAGS_TYPES = (
+        (BAN, "Email ban"),
+        (FLAG, "Email flag"),
+    )
+    email_address = models.EmailField(
+        help_text="Email address flagged.",
+    )
+    flag_type = models.CharField(
+        help_text="The flag type assigned.",
+        choices=FLAGS_TYPES,
+        max_length=5,
+    )
+    flag = models.CharField(
+        help_text="The flag assigned to email address.",
+        max_length=25,
+        blank=True,
+    )
+    reason = models.CharField(
+        help_text="The notification subtype",
+        max_length=50,
+        blank=True,
+    )
```

Comment:
Done! I added a class `SUB_TYPES` to handle these choices, also changed the field name from `reason` to `event_sub_type` because we are going to use for bounces and complaints subtypes